site/src/request_handlers/github.rs

```diff
@@ -163,6 +160,68 @@ async fn handle_rust_timer(
     Ok(github::Response)
 }
 
+/// Parses the first occurrence of a `@rust-timer queue <...>` command
+/// in the input string.
+fn parse_queue_command(body: &str) -> Option<Result<QueueCommand, String>> {
+    let prefix = "@rust-timer";
+    let bot_line = body.lines().find_map(|line| {
+        let line = line.trim();
+        line.find(prefix)
+            .map(|index| line[index + prefix.len()..].trim())
+    })?;
+
+    let args = bot_line.strip_prefix("queue").map(|l| l.trim())?;
```

Comment:
GH never reformats comments, right? That is, it will not introduce unexpected `\n`s that would separate the command from some of its arguments?

Reply:
Uhh, that would be quite surprising I think. GH actually has a bunch of formats of the comments, like text, HTML, MD etc., but I think (hope) that here we work with the raw text. We were already requiring `=` to be right after `include` etc. before, so I think that this should be fine.